var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -70,6 +70,10 @@ class Qt(Package):
             description="Build with OpenSSL support.")
     variant('freetype', default='spack', description='Freetype2 support',
             values=('spack', 'qt', 'none'), multi=False)
+    variant('gold', default=False,
+            description='Use gold linker')
+    variant('xcb', default='spack', description='XCB support',
```

Comment:
I would make this Boolean, either `False` (no xcb support) or `True` (depend on Spack package)